lib.php

```diff
@@ -466,15 +466,15 @@ public function print_disclosure($cmid) {
             return html_writer::tag(
                 'div',
                 get_string('eulaalreadyaccepted', 'plagiarism_turnitinsim'),
-                array('class' => 'eulacontainer', 'id' => 'eulacontainer')
+                array('class' => 'turnitinsim_eulacontainer', 'id' => 'turnitinsim_eulacontainer')
```

Comment:
Do we need to change the id as well?

Reply:
@dwinn Do you know if the Moodle standard for prefixing html containers with the plugin name applies to both class and id?